src/libraries/System.Text.Json/tests/Resources/Strings.resx

```diff
@@ -171,7 +171,7 @@ tiline\"another\" String\\"],"str":"\"\""}</value>
f/dotnet/ru/System.Collections.xml","ref/dotnet/zh-hans/System.Collections.xml","ref/dotnet/es/System.Collections.xml","runtimes/win8-aot/lib/netcore50/System.Collections.dll","ref/MonoAndroid10/_._","ref/MonoTouch10/_._","ref/net46/_._","ref/xamarinios10/_._","ref/xamarinmac20/_._","package/services/metadata/core-properties/b4f8061406e54dbda8f11b23186be11a.psmdcp","[Content_Types].xml"]},"System.Collections.Concurrent/4.0.10":{"sha512":"ZtMEqOPAjAIqR8fqom9AOKRaB94a+emO2O8uOP6vyJoNswSPrbiwN7iH53rrVpvjMVx0wr4/OMpI7486uGZjbw==","type":"Package","files":["_rels/.rels","System.Collections.Concurrent.nuspec","lib/dotnet/System.Collections.Concurrent.dll","lib/MonoAndroid10/_._","lib/MonoTouch10/_._","lib/net46/_._","lib/xamarinios10/_._","lib/xamarinmac20/_._","ref/dotnet/System.Collections.Concurrent.dll","ref/dotnet/System.Collections.Concurrent.xml","ref/dotnet/zh-hant/System.Collections.Concurrent.xml","ref/dotnet/de/System.Collections.Concurrent.xml","ref/dotnet/fr/System.Collections.Concurrent.xml","ref/dotnet/it/System.Collections.Concurrent.xml","ref/dotnet/ja/System.Collections.Concurrent.xml","ref/dotnet/ko/System.Collections.Concurrent.xml","ref/dotnet/ru/System.Collections.Concurrent.xml","ref/dotnet/zh-hans/System.Collections.Concurrent.xml","ref/dotnet/es/System.Collections.Concurrent.xml","ref/MonoAndroid10/_._","ref/MonoTouch10/_._","ref/net46/_._","ref/xamarinios10/_._","ref/xamarinmac20/_._","package/services/metadata/core-properties/c982a1e1e1644b62952fc4d4dcbe0d42.psmdcp","[Content_Types].xml"]},"System.Collections.Immutable/1.1.37":{"sha512":"fTpqwZYBzoklTT+XjTRK8KxvmrGkYHzBiylCcKyQcxiOM8k+QvhNBxRvFHDWzy4OEP5f8/9n+xQ9mEgEXY+muA==","type":"Package","files":["_rels/.rels","System.Collections.Immutable.nuspec","lib/dotnet/System.Collections.Immutable.dll","lib/dotnet/System.Collections.Immutable.xml","lib/portable-net45+win8+wp8+wpa81/System.Collections.Immutable.xml","lib/portable-net45+win8+wp8+wpa81/System.Collections.Immutable.dll","package/services/metadata/core-properties/a02fdeabe1114a24bba55860b8703852.psmdcp","[Content_Types].xml"]},"System.Collections.NonGeneric/4.0.0":{"sha512":"rVgwrFBMkmp8LI6GhAYd6Bx+2uLIXjRfNg6Ie+ASfX8ESuh9e2HNxFy2yh1MPIXZq3OAYa+0mmULVwpnEC6UDA==","type":"Package","files":["_rels/.rels","System.Collections.NonGeneric.nuspec","lib/dotnet/System.Collections.NonGeneric.dll","lib/net46/System.Collections.NonGeneric.dll","lib/MonoAndroid10/_._","lib/MonoTouch10/_._","lib/xamarinios10/_._","lib/xamarinmac20/_._","ref/dotnet/System.Collections.NonGeneric.dll","ref/dotnet/System.Collections.NonGeneric.xml","ref/dotnet/zh-hant/System.Collections.NonGeneric.xml","ref/dotnet/de/System.Collections.NonGeneric.xml","ref/dotnet/fr/System.Collections.NonGeneric.xml","ref/dotnet/it/System.Collections.NonGeneric.xml","ref/dotnet/ja/System.Collections.NonGeneric.xml","ref/dotnet/ko/System.Collections.NonGeneric.xml","ref/dotnet/ru/System.Collections.NonGeneric.xml","ref/dotnet/zh-hans/System.Collections.NonGeneric.xml","r
```

Comment:
This JSON test string change isn't necessary. Consider reverting.

Reply:
Is the change unimportant or incorrect here? 

I have replaced `System.IO.Compression.clrcompession`  --> `System.IO.Compression.Native` in other places.
I am mostly curious what this maps to.

Reply:
I would treat this string as a black-box test string for JSON. The test was produced from a copy of a project.lock.json file at some point-in-time for ensuring some of the build-tools/dotnet cli processing of this file works as expected.

The contents of the string shouldn't be relevant for the intended purpose of this PR, imo. At best, changing this string here is frivolous and hence an unnecessary change to introduce when looking back at git blame/history of the file.

Reply:
I see. I will revert the resource change then. I thought it needs to match the package name.